website_sale_brand_catalogs/tests/test_catalog.py

```diff
@@ -0,0 +1,38 @@
+from odoo.tests import HttpCase, tagged
+
+from odoo.addons.website.tools import MockRequest
+from odoo.addons.website_sale_brand_catalogs.controllers.portal import WebsiteCatalogs
+
+
+@tagged("post_install", "-at_install")
+class TestCatalog(HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.website = cls.env.ref("website.default_website")
+        cls.brand = cls.env["product.brand"].create({"name": "My Brand"})
+        cls.test_catalog = [("catalog_ids", "!=", False)]
+
+    def test_01_domain_catalogs(self):
+        catalogs = WebsiteCatalogs.get_domain_catalogs(self)
+        self.assertEqual(self.test_catalog, catalogs)
+
+    def test_02_brand_detail(self):
+        with MockRequest(self.env, website=self.website):
+            detail = WebsiteCatalogs().catalog_brand_detail(self.brand)
+            self.assertEqual(detail["brand"].name, self.brand.name)
+
+    def test_03_catalog_brands(self):
+        with MockRequest(self.env, website=self.website):
+            brand = WebsiteCatalogs().catalog_brands()
+            self.assertEqual(brand.name, self.brand.name)
+
+    def test_04_lead(self):
+        self.env["res.config.settings"].write(
```

Comment:
This is an empty recordset, a write shouldn't have any effect.

If you require that group for this module to work, maybe you should enable it on a `settings.xml` file